sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -923,7 +923,7 @@ class ContainerSasPermissions(object):
         Write a new blob, snapshot a blob, or copy a blob to a new blob.
     :keyword bool permanent_delete:
         To enable permanent delete on the blob is permitted.
-    :keyword bool find:
+    :keyword bool filter_by_tags:
```

Comment:
Is this going to be consistent with Storage reference docs?

Reply:
So the storage reference docs have this as 'find' and that's why I originally went with that BUT this 'f' permission already exists on an account SAS and we called it `filter_by_tags`. It is the same permission just here it is specific to a container.

What do you think the best thing to do here is? Match the existing code or match the [docs](https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas)

Reply:
In that case let's go with `filter_by_tags`. It will be consistent with the existing API, and also is more descriptive in terms of how it will be used/what permissions it's actually giving you :)